src/SignalRServiceExtension/SignalREndpointsAttribute.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Description;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    [AttributeUsage(AttributeTargets.Parameter)]
+    [Binding]
+    public class SignalREndpointsAttribute : Attribute
+    {
+        public string ConnectionStringSetting { get; set; } = Constants.AzureSignalRConnectionStringName;
+
+        [AutoResolve]
+        public string HubName { get; set; }
```

Comment:
Is `HubName` needed? Customer should configure Hub settings in required single place instead of in different bindings object. This setting looks support customer send to a different hub.

Reply:
To do in the next version